application/features/bootstrap/EmailContext.php

```diff
@@ -1193,35 +1211,19 @@ public function iSendPasswordExpiredEmails()
     }
 
     /**
-     * @Given a mfaManagerBcc email address is configured
-     */
-    public function aMfamanagerbccEmailAddressIsConfigured()
-    {
-        \Yii::$app->params['mfaManagerBcc'] = 'email@example.com';
-    }
-
-    /**
-     * @Then the mfaManagerBcc email address is on the bcc line
-     */
-    public function theMfamanagerbccEmailAddressIsOnTheBccLine()
```

Comment:
This test step is still in use on line 328 of `application/features/email.feature`, so we probably shouldn't delete it here (unless we also delete it there... but I think that decision might still be pending).